executor/runtime/docker/docker.go

```diff
@@ -449,6 +449,12 @@ func (r *DockerRuntime) mainContainerDockerConfig(c runtimeTypes.Container, bind
 		"/run/netns": "rw,size=" + defaultRunTmpFsSize,
 	}
 
+	// For performance reasons, we put the hsperfdata for JVM-based workloads on a tmpfs
+	// See https://www.evanjones.ca/jvm-mmap-pause.html for an explanation. JVM-67
+	hostCfg.Tmpfs["/tmp/hsperfdata_www-data"] = "rw,size=" + defaultRunTmpFsSize
```

Comment:
The blog post mentions that these are paths to files rather than directories, is that okay here in the context of creating mountpoints?

Reply:
Waaaat?
I was going based off of https://stash.corp.netflix.com/projects/BASEAMI/repos/nflx-ezconfig-core/browse/root/apps/ezconfig/templates/apps/ezconfig/gen/java_run.sh#67-81

cc @danielthomas

Reply:
OK it looks like the blog post is inaccurate because I checked `/tmp` and it's a directory:
```
$ ls -l /tmp
drwxr-xr-x 2 nickparker    100 Oct 12 17:55 hsperfdata_nickparker
```

So nevermind, good to go IMO unless @DanielThomas has anything

Reply:
Yeah, it's a directory. Each process gets a `<pid>` file in there.